docker-compose.test.yaml

```diff
@@ -69,6 +69,13 @@ services:
     environment:
       MYSQL_ROOT_PASSWORD: root
 
+  test-flask-vulnerable-api:
+    build:
+      context: .
+      dockerfile: test/data/flask_vulnerable_api/Dockerfile
+    ports:
+      - 5000:5000
```

Comment:
I don't think this directive is needed, this will exposed the port on the host - if you skip it, the port will still be exposed on the container (but not on the host)